src/pages/settings.tsx

```diff
@@ -1,24 +1,17 @@
 import styles from 'styles/pages/settings.module.scss'
-import Loader from 'components/Loader/Loader'
 import EditDisplayName from 'components/EditDisplayName/EditDisplayName'
-import { useUser } from 'hooks/useUser'
 import { withDefaultLayout } from 'layout/DefaultLayout/DefaultLayout'
 import { useEditDisplayNameMutation } from 'operations/portal/mutations/editDisplayName.g'
-import { useGetDisplayNameQuery } from 'operations/portal/queries/getDisplayName.g'
+import { useAuthContext } from 'stores/authContext'
 
 const Settings = () => {
-  const { user } = useUser()
+  const { portalUser } = useAuthContext()
 
   const [handleEditDisplayName] = useEditDisplayNameMutation()
 
-  const { error, data } = useGetDisplayNameQuery()
-  const userDisplayName = (data?.displayName || '') as string
+  const userDisplayName = (portalUser?.displayName || '') as string
```

Comment:
We're getting the user's display name in the `getUser` query now, so `useGetDisplayNameQuery` doesn't need to run here. Seemed to be a simple enough change to make on this branch.